src/test/java/de/tum/in/www1/artemis/ParticipationIntegrationTest.java

```diff
@@ -1158,4 +1165,9 @@ void getParticipation_submittedNotEndedQuizBatch() throws Exception {
         assertThat(actualSubmittedAnswerText.getText()).isEqualTo("test");
         assertThat(actualSubmittedAnswerText.isIsCorrect()).isFalse();
     }
+
+    private static Stream<Arguments> getGetParticipationsubmittedNotEndedQuizParameters() {
+        return Stream.of(Arguments.of(QuizMode.SYNCHRONIZED, true), Arguments.of(QuizMode.SYNCHRONIZED, false), Arguments.of(QuizMode.BATCHED, true),
+                Arguments.of(QuizMode.BATCHED, false), Arguments.of(QuizMode.INDIVIDUAL, true), Arguments.of(QuizMode.INDIVIDUAL, false));
+    }
```

Comment:
**(Optional)** I think it would be more readable code and show more clear what's going on if you format it this way.
```suggestion
    private static Stream<Arguments> getGetParticipationsubmittedNotEndedQuizParameters() {
        return Stream.of(
                Arguments.of(QuizMode.SYNCHRONIZED, true), 
                Arguments.of(QuizMode.SYNCHRONIZED, false), 
                Arguments.of(QuizMode.BATCHED, true),
                Arguments.of(QuizMode.BATCHED, false), 
                Arguments.of(QuizMode.INDIVIDUAL, true), 
                Arguments.of(QuizMode.INDIVIDUAL, false)
        );
    }
```

Reply:
I agree this is more readable, but the formatting was triggered from the git hook and the format is breaking the server-style CI. So I need to revert your suggestion for now, unfortunately.